src/components/ChecModal.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-  <component :is="form ? 'form' : 'div'" class="modal__overlay" @click.self="emitClose">
+  <component
+    :is="form ? 'form' : 'div'"
+    :class="classObject"
```

Comment:
Could've just made this:

```
    :class="`modal__overlay modal__overlay--${this.overlay}`"
```

but 🤷‍♂️ .

Reply:
Updated based on feedback